chickadee/processes/wps_bccaq.py

```diff
@@ -0,0 +1,143 @@
+from pywps import Process, ComplexInput, ComplexOutput, LiteralInput, FORMATS
+from pywps.app.Common import Metadata
+from netCDF4 import Dataset
+from rpy2.robjects.packages import isinstalled, importr
+
+from wps_tools.utils import log_handler
+from wps_tools.io import log_level
+from chickadee.utils import logger, set_r_options
+
+# Install and import R packages
+if not isinstalled("ClimDown"):
+    utils = importr("utils")
+    utils.install_packages("ClimDown")
```

Comment:
@nikola-rados or @nannau this is causing an error in the workflow tests. Is there some other way I should get R packages?

Reply:
Github actions and `R` and `python don't get along super well as @nannau and I have found out. Don't worry about actions, I can look into the devopsy stuff. Just make sure your tests pass with `make` locally so that we can repeat them.